Ensure that ancillary scripts in the same directory can be found when
chkrootkit is not run from its directory.
Has been reported to the developers by email.
--- chkrootkit.orig	2021-06-09 11:59:50.000000000 -0500
+++ chkrootkit	2022-03-22 14:05:29.000000000 -0500
@@ -10,6 +10,8 @@
 # (c)1997-2021 Nelson Murilo, Pangeia Informatica, AMS Foundation and others.
 # All rights reserved
 
+cd "$(dirname "$0")"
+
 ### workaround for some Bourne shell implementations
 unalias login > /dev/null 2>&1
 unalias ls > /dev/null 2>&1
